launcher/Application.cpp

```diff
@@ -701,6 +701,9 @@ Application::Application(int& argc, char** argv) : QApplication(argc, argv)
         m_settings->registerSetting("InstSortMode", "Name");
         m_settings->registerSetting("SelectedInstance", QString());
 
+        // Shortcut creation
+        m_settings->registerSetting("ShortcutCreationMode", "Desktop");
```

Comment:
Why not put that dialog inside the Right-click instance dialog similar to the Export action:
The default action would be the first action from the list but then expand to multiple options like you have right now on settings.
I do think is a bit to much to have a setting for it, as it looks like an action you can choose from.

Reply:
Oh, yeah that seems much simpler, just didn't think of that. Will implement